behat.yml.dist

```diff
@@ -16,8 +16,9 @@ default:
               user registration: '/user/register'
   extensions:
     Drupal\MinkExtension:
-      goutte: ~
-      selenium2: ~
+      goutte:
+        guzzle_parameters:
+          verify: false
```

Comment:
I made the test not to use Selenium (removed the `@javascript` tag). The test makes no JavaScript testing and should not be a JS test. But now, as is using Guzzle to make requests, that one is throwing `GuzzleHttp\Exception\RequestException` exceptions because the requests are HTTPS but the eCAS Mock Sever uses a self-signed certificate. See https://stackoverflow.com/questions/35637140/behat-selenium-tests-trust-self-signed-ssl-cerificate

Reply:
Oh right! That makes sense, thanks!